src/BloomExe/Collection/CollectionSettings.cs

```diff
@@ -396,6 +400,74 @@ public void Load()
 			{
 				//ah well, we tried, no big deal, only a couple of beta testers used this old name
 			}
+
+			// Check if we need to do a one time check (perhaps migrate to a new Settings value)
+			if (OneTimeCheckVersionNumber < _currentOneTimeCheckVersionNumber)
+			{
+				DoOneTimeCheck();
+			}
+		}
+
+		private void DoOneTimeCheck()
+		{
+			// If we ever have to do another one of these besides our minimal Andika New Basic migration
+			// we should refactor this so it calls a method based on the OneTimeCheckVersionNumber
+			do
+			{
+				if(!MigrateSettingsToAndikaNewBasicFont())
+					break; // in case of failed migration
+				OneTimeCheckVersionNumber++;
+
+			} while (OneTimeCheckVersionNumber < _currentOneTimeCheckVersionNumber);
```

Comment:
If I'm reading this right, it's going to perform the check even if it's already checked it before.  All that is needed here is

    if(OneTimeCheckVersionNumber < 1)
        if(MigrateSettingsToAndikaNewBasicFont())
            OneTimeCheckVersionNumber = 1;